### ISSUE TITLE
Feature Request: Make Unit-tests fail if the accessibitilty issue if found

### ISSUE THREAD

**Issue by ZimboPro** [rebelappstudio/accessibility_tools#89]
Thanks for this library. It has been a massive time saver.

It would be useful to be able to have this library check for accessibility issues during tests and fail with the accessibility issue as the reason.

Personally, I would like this feature to also be available as far back as 2.5.0 at least as well, as the current project is on an older Flutter version.

**Comment by ZimboPro** [comment]
Hi @aednlaxer 

To answer your question, yes. But I was not aware Flutter's accessibility guidelines for tests. That is perfect for my use case. The ticket can be closed if you think it is not suitable for this package
